frontend/src/libraries/explorer-wamp/receipts.ts

```diff
@@ -119,4 +119,22 @@ export default class ReceiptsApi extends ExplorerApi {
       throw error;
     }
   }
+
+  async isReceiptInTransactionIndexed(receiptId: string) {
```

Comment:
1. The type of the return value should be always clearly defined to check out expectations and let readers better understanding of the code without a need to read the implementation
2. The name of the function currently does not match the implementation `is*` usually imply that it will return a boolean, but it is not the case here, and we want to return the transaction hash here, so we should rename the function
3. Use the methods in explorer-wamp package to convert from snake case to camel case (`return { originatedFromTransactionHash: originated_from_transaction_hash }), so the rest of the code base can use JS code-style variable names